receiver/signalfxreceiver/README.md

```diff
@@ -1,5 +1,11 @@
 # SignalFx Receiver
 
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [beta]        |
```

Comment:
Could you please change it to `stable` instead?

Reply:
@pjanotti done